testsuite/forge/src/backend/local/mod.rs

```diff
@@ -129,22 +129,27 @@ impl LocalFactory {
         R: ::rand::RngCore + ::rand::CryptoRng,
     {
         let version = self.versions.keys().max().unwrap();
-        self.new_swarm_with_version(rng, number_of_validators, version)
+        self.new_swarm_with_version(rng, number_of_validators, version, None)
     }
 
     pub fn new_swarm_with_version<R>(
         &self,
         rng: R,
         number_of_validators: NonZeroUsize,
         version: &Version,
+        genesis_modules: Option<Vec<Vec<u8>>>,
     ) -> Result<LocalSwarm>
     where
         R: ::rand::RngCore + ::rand::CryptoRng,
     {
-        let mut swarm = LocalSwarm::builder(self.versions.clone())
+        let mut builder = LocalSwarm::builder(self.versions.clone())
             .number_of_validators(number_of_validators)
-            .initial_version(version.clone())
-            .build(rng)?;
+            .initial_version(version.clone());
+        if let Some(genesis_modules) = genesis_modules {
+            builder = builder.genesis_modules(genesis_modules);
```

Comment:
Rather than having to check `if Some` or use things like `unwrap_or_else` downstream, could we set a default earlier in the code path, and know confidently later on that we have a nonoption Vec?

ie:
```rust
let mut builder = LocalSwarm::builder(self.versions.clone())
            .number_of_validators(number_of_validators)
            .genesis_modules(genesis_modules.unwrap_or_else(|| diem_framework_releases::current_module_blobs().to_vec())
            .initial_version(version.clone());
```

Reply:
For now I think the decision for what the default should be is done in the correct place, specifically this lets us error out when using the k8s backend because support for that doesn't exist yet. Once that is done then I think we could tweak that.